features/flows/elastic/src/main/java/org/opennms/netmgt/flows/elastic/ElasticFlowRepository.java

```diff
@@ -194,7 +188,6 @@ public ElasticFlowRepository(MetricRegistry metricRegistry, JestClient jestClien
         this.indexSettings = Objects.requireNonNull(indexSettings);
 
         flowsPersistedMeter = metricRegistry.meter("flowsPersisted");
-        logConversionTimer = metricRegistry.timer("logConversion");
```

Comment:
We should remove graphs and associated JMX data-collection config too.

Reply:
done